cvcl/urls.py

```diff
@@ -28,4 +31,5 @@
     url(r'^envdefs/(?P<pk>\d+)/createvmdef$', VmDefinitionCreate.as_view(), name='envdefs.createvmdef'),  # instructor
     url(r'^vmdefinitions/(?P<pk>\d+)/update$', VmDefinitionUpdate.as_view(), name='vmdef.update'),  # instructor
     url(r'^vmdefinitions/(?P<pk>\d+)/delete$', VmDefinitionDelete.as_view(), name='vmdef.delete'),  # instructor
+    url(r'^courses/(?P<pk>\d+)/deleteStudent/(?P<user_pk>\d+)', DeleteStudent.as_view(), name='delete_student'),
```

Comment:
This line should be moved to line 26. Why did you make the order random?
Also, URL should be lowercase. Change class name to follow pattern and name too.